_data/cryptocurrencies.yml

```
@@ -406,6 +406,14 @@ websites:
       - totp
     doc: https://support.hitbtc.com/en/support/solutions/articles/63000224976
 
+  - name: HUOBI
+    url: https://huobi.com
```

Other URL points to an invalid certificate.
```suggestion
    url: https://www.huobi.com/
```